src/base/datetime/date/index.ts

```diff
@@ -282,43 +276,14 @@ export class BaseDate extends KucBase {
     dispatchCustomEvent(this, "kuc:base-date-change", detail);
   }
 
-  private _openCalendarByKeyCode() {
-    this._valueForReset = this.value;
-    this._openCalendar();
-    this._toggleEl.blur();
-  }
-
-  private _handleBlurButton() {
-    this._dateInput.classList.remove("kuc-base-date__input--focus");
```

Comment:
does we need class "kuc-base-date__input--focus" ?
If we don't use it anywhere, please remove it in src/base/datetime/date/style.ts